### DISCUSSION TITLE
have two questions

### DISCUSSION THREAD

**Discussion by realfasix** [slowscript/warpinator-windows#30] [Q&A] [answered]
Finally a real replacement of NitroShare! Great!
Said that, have two questions.
1. Why portable version 0.4 have languages but installer version not?
2. How clean transfer history?

Thanks in advance!

**Comment by slowscript** [comment, marked as answer]
Hi,

> Why portable version 0.4 have languages but installer version not?

Sorry about this, it was a mistake. The installer was configured incorrectly, so all the localized Warpinator.resources.dll files were placed into the main program directory instead of the correct subdirectory. I have added warpinator-setup_0.4r2.exe to the release page that fixes this.

> How clean transfer history?

Cleaning transfer history is not possible at the moment (apart from restarting the application). The other versions of Warpinator have this feature so I will add a button for this here as well, hopefully soon.
